### DISCUSSION TITLE
Issue getting Copilot to work

### DISCUSSION THREAD

**Discussion by adophilus** [zbirenbaum/copilot.lua#224] [General]
Hi everyone, sorry if this might sound stupid but I've been trying to setup GitHub Copilot on neovim since this afternoon. I started with the official github copilot neovim plugin but it didn't work. Then I tried this plugin but it's still not working. By 'not working' I mean this:
![image](https://github.com/zbirenbaum/copilot.lua/assets/45528869/6e562387-3ea1-4ff7-9f7a-415b49d7ae10)

It shows that the plugin is installed, but autostart is set to false and I don't see any ghost text in my editor.

Here's my neovim config for the plugin (I use astrovim -- using the lazy package manager with file based plugins):
![image](https://github.com/zbirenbaum/copilot.lua/assets/45528869/5de66238-0960-453b-a722-4d7b5dcd01d6)

**Comment by MunifTanjim** [comment]
What does `:Copilot status` output?

**Reply by adophilus** [reply]
Sorry for the late response. It's working now. Not because I fixed it though, but because I switched to linux 
![2023-11-08-19:39:47](https://github.com/zbirenbaum/copilot.lua/assets/45528869/b5f1d80d-91ab-4748-8f3f-27b57f61134e)
